index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Ejemplo html </h1>
+    <button>click aquí</button>
+</body>
+</html>
```

Comment:
You can add break here in the future :)
```suggestion
</html>

```